advanced/am-pattern-1/confs/am/instance-2/repository/conf/deployment.toml

```diff
@@ -151,78 +76,16 @@ allow_credentials = false
 enable_data_publishing = false
```

Comment:
@Chathuru please check the previous comment with regards to data publishing.

Reply:
Fix in https://github.com/Chathuru/kubernetes-apim/commit/0205f6311ce5b9ff81ebe108aa79ff7a040078e1 commit